ts-cloudfs/README.md

```diff
@@ -0,0 +1,65 @@
+# Cloud FS Sample App
+
+## Overview 
+
+The `ts-cloudfs` sample application demonstrates how to use the `@klotho::persist` annotation to store secrets. The sample application exposes one endpoint, with path `/` to return the content of the stored secret.
+
+## Prerequisites
+
+This guide assumes:
+- `ts-node` and `tsc` installed globally OR `./node_modules/bin` is on the `PATH`.
```

Comment:
Can we replace this with using `npx ts-node` / `npx tsc`?

Reply:
yeah just the entire line right? 

wanted to shy away from global installation

Reply:
actually it shouldnt even be a prereq because the npm i will install this ability so i can remove this